admin/server_info.php

```diff
@@ -113,7 +116,7 @@
         ?>
         <tr class="db-row">
             <td class="db-info db-name"><?php echo $variable['Variable_name']; ?></td>
-            <td class="db-info db-value"><?php echo empty($variable['Value']) ? '$nbsp;' : $variable['Value']; ?></td>
+            <td class="db-info db-value"><?php echo empty($variable['Value']) ? '$nbsp;' : htmlspecialchars($variable['Value']); ?></td>
```

Comment:
why did you remove the call to `htmlspecialchars`?

Reply:
I didn't I added a call to htmlspecialchars

Reply:
sorry I misread this.